chirp/drivers/ga510.py

```diff
@@ -1102,9 +1106,9 @@ class RadioddityGS5BRadio(Senhaix8800Radio):
 
 
 @directory.register
-class SignusXTR5Radio(Senhaix8800Radio):
-    """Signus XTR-5"""
-    VENDOR = "Signus"
+class CignusXTR5Radio(Senhaix8800Radio):
+    """Cignus XTR-5"""
+    VENDOR = "Cignus"
```

Comment:
You can't just change a vendor/model like this as you will break compatibility with everyone's existing images for this radio. Was this really a typo (if so, that's frustrating) or does this radio exist with both spellings? Either way, we can't change it without accounting for the change in some way to avoid breaking users. There's a compatibility list in the loader, which we can use if we have to, but if it's spelled the old way in some places, we should just leave it.